docs/guide/migrationV2.md

```diff
@@ -13,6 +13,14 @@ A number of changes were made to the configuration options passed to the plugin
  * `xPadding` and `yPadding` options were merged into a single `padding` object in the label configuration of line annotation to align with Chart.js options.
  * `enabled` option was replaced by `display` in the callout configuration of label annotation, in the label configuration of line and box annotations and in the arrow heads configuration of line annotation to have the same option on all elements.
 
+## Elements
+
+`chartjs-plugin-annotation` plugin version 2 normalizes the properties of the annotation elements in order to be based on common box model, as following:
+
+![145609678-792aac72-2cf0-4bb7-9b05-464a605e8d80](https://user-images.githubusercontent.com/11741250/156187206-62cb4236-527e-45ae-9dc6-6b6c50ad8991.png)
+
+Additional properties are stored for specific annotation types:
```

Comment:
Seems like you wanted to list things here but forgot to write the list or forgot to remove the `:` :)

Reply:
> Seems like you wanted to list things here but forgot to write the list or forgot to remove the `:` :)

yes... I'll remove it. Thank youuuu!!!

Reply:
done

Reply:
@LeeLenaleee what do you think about the image? I mean, the image is uploaded to user-images.githubusercontent.com and not present in the project because I used in the PR and issue. I don't know if the images loaded there are available forever. make sense to move it in the project?

Furthermore, do you think could make sense to have a picture for each annotation type? Some properties are not described here (the reason of the uncompleted sentence), because there are not common and not described in the type as well.

Reply:
Guess it would be better to just include it in the project. So people that clone it can more easily acess and see it.

Haven't read through the rest of this PR so dont really know the implementation but if it is different enough that you need to use it differently I think it is a good thing to also add the other versions as images. Can better have a few too many to explain as too little and it not being clear for users

Reply:
> Guess it would be better to just include it in the project. So people that clone it can more easily acess and see it.

make sense!



> Haven't read through the rest of this PR so dont really know the implementation but if it is different enough that you need to use it differently I think it is a good thing to also add the other versions as images. Can better have a few too many to explain as too little and it not being clear for users

make sense as well.